migration/migration.go

```diff
@@ -640,10 +640,10 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB) error {
 				"err":  err,
 				"id":   t.Template.ID,
 				"name": t.Template.Name,
-			}, `failed to import space template #%d with name "%s" and ID %s`, idx, t.Template.Name, t.Template.ID)
-			return errs.Wrapf(err, `failed to import space #%d template with name "%s" ID %s`, idx, t.Template.Name, t.Template.ID)
+			}, `failed to import space template #%d with name '%s' and ID %s`, idx, t.Template.Name, t.Template.ID)
```

Comment:
Why that change?

Reply:
the error comes up in double quotes already and having there nested double quote in there the output looks weird because it gets escaped by back slashes, so to make the output cleaner and readable i used single quotes which is not escaped inside double quotes.

Reply:
But the final consumer doesn't have the double quotes.